### DISCUSSION TITLE
Delegating Tab Improvements

### DISCUSSION THREAD

**Discussion by joesixpack** [subquery/subql#986] [Frontier TestNet - Bug Reporting]
The Delegating tab only shows you your current delegations and an Undelegate button, but there is no way to delegate more tokens to any of the Indexers listed.  Suggest each Indexer become clickable to their profile page so the Delegate button can be reached.

**Comment by mzxyz** [comment]
@lilian79511

**Comment by joesixpack** [comment]
Similar to an Indexer's own node showing at the top, maybe moving the Indexers that have been Delegated to to the top of the Indexers list is a good idea as well.
